core/src/labels/label.cpp

```diff
@@ -132,8 +131,11 @@ void Label::enterState(State _state, float _alpha) {
 }
 
 void Label::setAlpha(float _alpha) {
-    m_transform.state.alpha = CLAMP(_alpha, 0.0, 1.0);
-    m_dirty = true;
+    float alpha = CLAMP(_alpha, 0.0, 1.0);
+    if (m_transform.state.alpha != alpha) {
```

Comment:
I'm not sure values between 0.0 and 1.0 changes anything, since mostly the fract part matters. This link is interesting though. The bit comparison failed several times for me with equal values:
![screen shot 2015-07-27 at 5 13 55 pm](https://cloud.githubusercontent.com/assets/7061573/8917921/0cc93966-3483-11e5-8613-e7c01b329f5e.png)